src/js/main.js

```diff
@@ -51,8 +59,12 @@ var SupportKit = Marionette.Object.extend({
 
     initialize: function() {
```

Comment:
The same API has both init and initialize ಠ_ಠ

I'd much prefer to have our SupportKit global stay "clean" and expose only public API, without our internal _foo stuff and Marionette.Object stuff.

Eg, have this guy be `var Main = Marionette.Object.extend({`

and add a separate supportkit.js wrapping `Main` that exposes `init` `login` etc..